services/datamanager/builtin/capture_test.go

```diff
@@ -29,8 +29,9 @@ var (
 	enabledBinaryCollectorConfigPath            = "services/datamanager/data/robot_with_cam_capture.json"
 	infrequentCaptureTabularCollectorConfigPath = "services/datamanager/data/fake_robot_with_infrequent_capture.json"
 	remoteCollectorConfigPath                   = "services/datamanager/data/fake_robot_with_remote_and_data_manager.json"
-	emptyFileBytesSize                          = 100 // size of leading metadata message
-	captureInterval                             = time.Millisecond * 10
+	// size of leading metadata message. Dan: Where did this come from? I'm observing 97 byte files.
```

Comment:
Should be reproducible if one runs `TestFilePolling` and looks at the filesize of the (IIRC) 3rd file.

Reply:
This was based on the size of the leading metadata message that was written, which is brittle since the contents of that can change based on additions/removals of fields from it. Going under like you have here is better.